src/routes/routes.js

```diff
@@ -6,13 +6,15 @@ import {
 } from 'react-router-dom';
 
 // @load component
-import Login from '../components/users/login';
 import Article from '../components/articles/AllArticles';
+import Auth from '../components/Auth/Auth';
 
 const Routes = () => (
   <Router>
     <Switch>
-      <Route exact path="/" component={Login} />
+      <Route exact path="/" component={Auth} />
```

Comment:
This route should be displaying the home component

Reply:
That is true,